intelligent-video-recording/config_detection.cfg

```diff
@@ -0,0 +1,15 @@
+# Use absolute path for model location and output file
+# Only MP4 recording is supported by the demo
+[DEMO_SETTINGS]
+CAMERA_DEVICE = /dev/video0
+CAMERA_DEVICE_WIDTH = 1280
+CAMERA_DEVICE_HEIGHT = 720
+ARCH=mobilenetv2ssd
+MODEL_LOCATION = /home/coral/models/ssd_mobilenet_v2_coco_quant_postprocess_edgetpu.tflite
+INPUT_LAYER = 'input'
+OUTPUT_LAYER = 'MobilenetV2/Predictions/Reshape_1'
+LABELS = coco_labels.txt
+OUTPUT_FILE = /home/coral/recordings/recording.mp4
+CLASSES_ID = [74]
```

Comment:
I think we should explain better what these numbers are in the README file. We should assume that people don't know that a number corresponds to a label and corresponds to the kind of object we will be tracking